test/e2e/scaffold/ingress.go

```diff
@@ -78,7 +78,7 @@ spec:
               port: 8080
             timeoutSeconds: 2
           image: "viewking/apisix-ingress-controller:dev"
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
It will be too slow if using `Always` policy.

Reply:
In order to prevent the mirror from already existing in other namespaces in the local environment.
For me, I have an `e2e` ns contains ingress running always.

Reply:
Even so, this should be an option, since in CI environment, we don't have this issue.